test/factories/memberships.rb

```diff
@@ -3,5 +3,9 @@
     association :user
     association :group
     membership_start { '2010-02-02' }
+
+    trait :leader do
+      posts { create_list(:post, 1, :leader) }
```

Comment:
I think if it's just 1 item then it can be:
```suggestion
      posts << create(:post, :leader)
```

Reply:
It is true, it should work, I've used like that at some other place.

I was thinking about which one is more expressive, maybe this one is a bit more, so I left it like this here.